### PR TITLE
Analytics Pull Request

### PR DESCRIPTION
**Pull request by stephentmoon**

### CODE REVIEW

File: queries/PublicationCount.rq [hunk 1]
```diff
@@ -1,9 +1,8 @@
 PREFIX bibo: <http://purl.org/ontology/bibo/>
 PREFIX dco: <http://info.deepcarbon.net/schema#>
 
-#
 #Counts the total number of publications on the DCO website
-#
+#Stephen Moon, July 2015
 SELECT (concat("Total Publications: ", (COUNT(?publication))) as ?number) WHERE
```
Comment (mrpatrickwest) [inline queries/PublicationCount.rq]: Do we need a distinct in front of ?publication

File: xslt/xml-to-html-statistics.xslt [hunk 2]
```diff
@@ -18,7 +18,7 @@ Version 1 : Dave Beckett (DAWG)
 Version 2 : Jeen Broekstra (DAWG)
 Customization for SPARQler: Andy Seaborne
 Customization for TWC: Patrick West
-Modified by Stephen Moon for DCO
+Modified by Stephen Moon for DCO Statistics
```
Comment (mrpatrickwest) [inline xslt/xml-to-html-statistics.xslt:24]: Is the header template even being called? If not then let's remove it.

File: queries/PublicationCount.rq (hunk 1 above)
Comment (stephentmoon) [inline queries/PublicationCount.rq]: Just tested it out, both give the same result (1532 publications).

On Mon, Jul 20, 2015 at 11:16 AM, Patrick West notifications@github.com
wrote:

> In queries/PublicationCount.rq
> https://github.com/tetherless-world/dco-twsparql/pull/2#discussion_r35005599
> :
> 
> >  #Counts the total number of publications on the DCO website
> > -#
> > +#Stephen Moon, July 2015
> >  SELECT (concat("Total Publications: ", (COUNT(?publication))) as ?number) WHERE
> 
> Do we need a distinct in front of ?publication
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/tetherless-world/dco-twsparql/pull/2/files#r35005599.

## 

Stephen T. Moon
(860) 480-8751

Comment (mrpatrickwest) [inline queries/PublicationCount.rq]: Go ahead and add distinct. Might not matter now, but could later. Just to be on the safe side.

File: xslt/xml-to-html-statistics.xslt [hunk 3]
```diff
@@ -119,7 +110,6 @@ Modified by Stephen Moon for DCO
 
   <xsl:template match="res:sparql">
 	<xsl:if test="res:head/res:link">
```
Comment (mrpatrickwest) [inline xslt/xml-to-html-statistics.xslt]: Remove the empty if statement

File: xslt/xml-to-html-statistics.xslt [hunk 4]
```diff
@@ -32,15 +32,6 @@ Modified by Stephen Moon for DCO
...
 
-  <xsl:template name="header">
-    <div>
-      <h2>Header</h2>
-      <xsl:for-each select="res:head/res:link"> 
-	<p>Link to <xsl:value-of select="@href"/></p>
-      </xsl:for-each>
-    </div>
-  </xsl:template>
-
```
Comment (mrpatrickwest) [inline xslt/xml-to-html-statistics.xslt:43]: below here, line 58 or so, there's a td tag that needs to be removed along with its ending tag. We're not using tables in this xslt

Comment (mrpatrickwest) [inline xslt/xml-to-html-statistics.xslt:43]: also below about line 46 in the vb-result template there's a div tag that needs to be removed and its ending tag. We don't need that either. We can always add it to the page if we'd like to.

**Pull request merged**